assets/src/edit-story/components/canvas/karma/quickActions.karma.js

```diff
@@ -52,7 +52,6 @@ describe('Quick Actions integration', () => {
       await fixture.events.click(await zoomSelector.option('Fill'));
       await fixture.events.sleep(300);
 
-      expect(fixture.screen.queryByRole('dialog')).toBeNull();
```

Comment:
🤔 hmm wonder if this is because we're using the context menu for right click too. Not a big deal since the next line is the same check, so this was redundant anyways.

Reply:
Or maybe the `dialog` role is also used in the popup

Reply:
yeah it was finding the checklist dialog since it stays in the DOM so we can keep counting issues. so that check is never going to come back as null anymore, it was until i removed the flag. We would need to specify the quickActions dialog, but like you said, feels redundant.